analyzers/src/SonarAnalyzer.CSharp/Rules/Utilities/TokenTypeAnalyzer.cs

```diff
@@ -50,9 +50,6 @@ public TokenClassifier(SyntaxToken token, SemanticModel semanticModel)
             protected override bool IsKeyword(SyntaxToken token) =>
                 SyntaxFacts.IsKeywordKind(token.Kind());
 
-            protected override bool IsContextualKeyword(SyntaxToken token) =>
-                SyntaxFacts.IsContextualKeyword(token.Kind());
```

Comment:
This one is removed with no replacement. `IsKeyword` invokes it already on Roslyn side so it's useless.